Gruntfile.js

```diff
@@ -1036,29 +1036,6 @@ module.exports = function(grunt) {
 				}
 			}
 		},
-		jsvalidate:{
```

Comment:
There are a few other places in the Gruntfile that references jsvalidate. We should remove all of these instances as well.

Reply:
Done